README.md

```diff
@@ -16,6 +16,26 @@ The default values are used internally, but might to be adjusted for your specif
 
 Funlen is included in [golangci-lint](https://github.com/golangci/golangci-lint/). Install it and enable funlen.
 
+## Configuration
+
+Available configuration options:
+
+```yaml
+linters-settings:
+  funlen:
+    # Checks the number of lines in a function.
+    # If lower than 0, disable the check.
+    # Default: 60
+    lines: -1
```

Comment:
Maybe set the values to the default settings instead of the disabled ones

Reply:
Makes sense. I just copied the entry from the golangci-lint examples.